### PR TITLE
Revamp readmes

### PR DESCRIPTION
**Pull request by aofarrel**
* Fix a dead link (readme.md =/= README.md)
* Clarify that I'm the point of contact for this repo, not the author of everything ever
* Add "advice" to HPC users, that advice being 🤷 
* Remove some excessive waffling from the readmes
* Show off some more features

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
 
 [![WDL 1.0 shield](https://img.shields.io/badge/WDL-1.0-lightgrey.svg)](https://github.com/openwdl/wdl/blob/main/versions/1.0/SPEC.md)  
-This is a work-in-progress project to implement some components of the University of Washington [TOPMed pipeline](https://github.com/UW-GAC/analysis_pipeline) into Workflow Description Language (WDL) in a way that closely mimics [the CWL version of the UW Pipeline](https://github.com/UW-GAC/analysis_pipeline_cwl). In other words, this is a WDL that mimics a CWL that mimics a Python pipeline. All three pipelines use the same underlying R scripts which do most of the heavy lifting, making their results directly comparable.
+This project is a Workflow Description Language (WDL) implementation of several components of the University of Washington [TOPMed pipeline](https://github.com/UW-GAC/analysis_pipeline), purposely done in a way that closely mimics [the CWL version of the UW Pipeline](https://github.com/UW-GAC/analysis_pipeline_cwl). In other words, this is a WDL that mimics a CWL that mimics a Python pipeline. All three pipelines use the same underlying R scripts which do most of the actual analysis, making their results directly comparable.
```
Comment (bethsheets) [inline README.md:4]: Suggest adding as last sentence: "We have also used checker-workflows to verify that results are scientifically equivalent."

**Review by bethsheets — APPROVED** [review]
One minor suggestion. Thanks!!

File: README.md [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
 
 ## Features
 * This pipeline is very similar to the CWL version, and while the main differences between the two [are documented](https://github.com/DataBiosphere/analysis_pipeline_WDL/blob/main/_documentation_/for%20users/cwl-vs-wdl-user.md), testing indicates they are functionally equivalent -- so much so that files generated by the CWL are used as truth files for the WDL   
 * As it works in a Docker container, it does not have any external dependencies other than the usual setup required for [WDL](https://software.broadinstitute.org/wdl/documentation/quickstart) and [Cromwell](http://cromwell.readthedocs.io/en/develop/)
 * Contains multiple checker workflows for validating sets of known inputs and expected outputs
+* Open-access sample data is provided, based upon sample data provided by UWGAC, itself based upon 1000 Genomes data
```
Comment (avani-k) [inline README.md:10]: Is the sample data we provide "based" upon the sample data provided by UWGAC? Or is it basically the same? I don't remember. If it is so, maybe reframe it to reflect that?

Suggestion:

Open-access sample data is provided, based upon sample data provided by UWGAC. This sample data is based upon the 1000 Genomes data

File: README.md [hunk 3]
```diff
@@ -15,16 +19,19 @@ Essentially all workflows which take in chromosome-level files share filename re
...
+### Running on your local machine
+Much preliminary testing and development of these pipelines was done by running Cromwell in "local mode," but we do not recommend this approach for doing actual analysis. Cromwell does not manage resources well on local executions. As a result, these pipelines (LD pruning especially) may get their processes killed by your OS and/or lock up Docker, even if running on downsampled data. These issues can *generally* be avoided by changing the concurrent job limit in your Cromwell configuration, and if you set this limit, you should find that all of the sample data in this repo should run on any Cromwell-and-Docker compatiable machine. [See instructions here](https://docs.dockstore.org/en/develop/getting-started/getting-started-with-wdl.html#setting-up-the-dockstore-cli) for how to set the concurrent job limit in the Dockstore CLI.
+
+### Running on an HPC
+These workflows have not been extensively tested in an HPC environment, but provided your HPC supports Cromwell and Docker, they should work as expected. You may which to run the checker workflows before doing actual analysis to ensure everything is running smoothly.
```
Comment (avani-k) [inline README.md:29]: "You may which to run the checker workflows before doing actual analysis to ensure everything is running smoothly."

Missing verb? Or something. This sentence feels off!

**Review by avani-k — APPROVED** [review]
Added a few minor comments. Looks good!

File: README.md (hunk 2 above)
Comment (aofarrel) [reply README.md:10]: Some of the data is modified. For example, UWGAC's copy of either chrX or chr1 (can't remember) has malformed data inside of it and breaks on either vcf to gds or ld pruning (again, can't remember) in both CWL and WDL form.

File: README.md (hunk 3 above)
Comment (aofarrel) [reply README.md:29]: Oops, that "which" was supposed to be "wish", thanks for spotting this!

**Pull request merged**